app/controllers/api/v1/orders_controller.rb

```diff
@@ -119,14 +120,21 @@ def order_record
           @order.created_by = current_user
           @order.detail_type = "GoodCity"
         end
+
+        if order_params['beneficiary_attributes'] and  @order.beneficiary and @order.beneficiary.created_by == nil
```

Comment:
You can use `@order.beneficiary.created_by.nil?`

Reply:
How about using code: `order_params['beneficiary_attributes'] and  @order.beneficiary.try(:created_by).nil?`

CC: @Darshan-Chauhan @steveyken